app/src/main/java/net/squanchy/analytics/Analytics.kt

```diff
@@ -81,12 +82,26 @@ class Analytics internal constructor(
 
     fun trackNotificationsEnabled() {
         firebaseAnalytics.setUserProperty("notifications_status", "enabled")
+        firebaseAnalytics.logEvent("notifications_status_changed", flagEnabled("true"))
```

Comment:
Can we enclose `"true"` and `"false into constants? Sooner or later I'm pretty sure we will misspell one of them

Reply:
👍 3caf33e23c5ed7316ea73833bf97100b14e12d9f